docs/guide/building-an-advanced-scene.md

```diff
@@ -4,238 +4,456 @@ type: guide
+The scene we will build is a **360&deg; image viewer**. There will be three
+panels which the user can click on. Once clicked, the background will fade and
+swap the 360&deg; images.
 
-This creates a basic static scene where the enemy stares at you even as you move around. We can use A-Frame components from the ecosystem to do some neat things.
+<!--toc-->
 
-## Using Components
+## Skeleton
 
-The [awesome-aframe repository][awesome] is a great place to find components that the community has created to enable new features. Many of these components are started from the [Component Boilerplate][boilerplate] and should provide builds in the `dist/` folders in their repositories. Take the [layout component][layout] for example. We can grab the build, drop it into our scene, and immediately be able to use a 3D layout system to automatically position entities. Instead of having one enemy, let's have ten enemies positioned in a circle around the player:
+This is the starting point for our scene:
 
-<p data-height="500" data-theme-id="0" data-slug-hash="bpNPjp" data-default-tab="html" data-user="mozvr" class="codepen">See the Pen <a href="http://codepen.io/team/mozvr/pen/bpNPjp/">Laser Shooter - Step 2</a> by MozVR (<a href="http://codepen.io/team/mozvr">@mozvr</a>) on <a href="http://codepen.io">CodePen</a>.</p>
+```html
+<a-scene>
+  <a-assets>
+    <!-- Images. -->
+    <img id="city" src="img/city.jpg">
+    <img id="city-thumb" src="img/thumb-city.png">
+    <img id="cubes" src="img/cubes.jpg">
+    <img id="cubes-thumb" src="img/thumb-cubes.png">
+    <img id="sechelt" src="img/sechelt.jpg">
+    <img id="sechelt-thumb" src="img/thumb-sechelt.png">
+  </a-assets>
+
+  <!-- 360-degree image. -->
+  <a-sky id="image-360" radius="10" src="#city"></a-sky>
+
+  <!-- Link. -->
+  <a-plane class="link" height="1" width="1"></a-plane>
+
+  <!-- Camera + Cursor. -->
+  <a-camera>
+    <a-cursor id="cursor">
+      <a-animation begin="cursor-click" easing="ease-in" attribute="scale"
+                   fill="backwards" from="0.1 0.1 0.1" to="1 1 1" dur="150"></a-animation>
+      <a-animation begin="cursor-fusing" easing="ease-in" attribute="scale"
+                   from="1 1 1" to="0.1 0.1 0.1" dur="1500"></a-animation>
+    </a-cursor>
+  </a-camera>
+</a-scene>
+```
 
-It is messy in markup to have the enemy entity duplicated ten times. We can drop in the [template component][template] to clean that up. We can also use the [animation system][animation] to have enemies march in a circle around us:
+We have predefined:
 
-<p data-height="500" data-theme-id="0" data-slug-hash="JXoQBm" data-default-tab="html" data-user="mozvr" class="codepen">See the Pen <a href="http://codepen.io/team/mozvr/pen/JXoQBm/">Laser Shooter - Step 3</a> by MozVR (<a href="http://codepen.io/team/mozvr">@mozvr</a>) on <a href="http://codepen.io">CodePen</a>.</p>
+- Several images to choose from in the [Asset Management System][ams] within `a-assets`.
```

Comment:
these bullet points end with periods but [the items in the numbered list above](https://github.com/aframevr/aframe/pull/1494/files#diff-152a0cc3c6c095bc8f95916fa320645dR16) don't; we should pick one